Description: fill in build.properties to fit for path of saxon and docbook-xsl
--- a/build.properties
+++ b/build.properties
@@ -13,7 +13,8 @@
 # Set this to the directory containing the DocBook Modular XSL Stylesheets
 #  from http://docbook.sourceforge.net/projects/xsl/
 
-xsl.stylesheet.home	=${local.software.home}/docbook-xsl-1.76.1
+#xsl.stylesheet.home	=${local.software.home}/docbook-xsl-1.76.1
+xsl.stylesheet.home    =/usr/share/xml/docbook/stylesheet/docbook-xsl
 
 # Set this to the directory containing the svnant Ant jars.  These are
 # needed for the srcdist target and to get the svn revision number
@@ -25,7 +26,8 @@
 # Set this to the directory where Saxon (http://saxon.sourceforge.net/)
 # is installed.
 
-saxon.home		=${local.software.home}/saxon-6.5.5
+#saxon.home		=${local.software.home}/saxon-6.5.5
+saxon.home     =/usr/share/java
 
 # Set this to the directory where FOP (http://xml.apache.org/fop/)
 # is installed.  (Note: this is disabled for now.)
